apps/module-smartcards/fixtures/tmp/long.b64

```diff
@@ -0,0 +1 @@
+VlgCucAvYoofoykT0QUEBmAeoA==
```

Comment:
did you mean to remove this and the next file? I'm assuming so since its under a tmp directory

Reply:
Thanks, removed.